tests/watchdog_threadpool_ut/watchdog_threadpool_ut.c

```diff
@@ -33,18 +33,21 @@ static void on_umock_c_error(UMOCK_C_ERROR_CODE error_code)
     ASSERT_FAIL("umock_c reported error :%" PRI_MU_ENUM "", MU_ENUM_VALUE(UMOCK_C_ERROR_CODE, error_code));
 }
 
-static void expect_init_verbose(int threapool_open_result)
+static void expect_init_verbose(int threadpool_result)

```

Comment:
>int threadpool_result [](http://example.com/codeflow?start=32&length=21)

This would be better making it a "bool fail_threadpool_create"
It would be more accurately representing what is happening.

Reply:
renames, rewrote some logic here.

Reply:
also rearranged `{`, very important.